index.html

```diff
@@ -114,7 +114,38 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-	 
+
+	 <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
It would be easier to identify your card, if you change this line and the last line to 
`<!-- ________ Chao's card START ________ -->`
and 
`<!-- ________ Chao's card END ________ -->`